database/seeder.go

```diff
@@ -0,0 +1,105 @@
+package database
+
+import (
+	"fmt"
+	"log"
+	"reflect"
+
+	"github.com/goravel/framework/contracts/database/seeder"
+)
+
+type Seeder struct {
+	Called []string
+}
```

Comment:
S: It seems we don't need this. We can use `facades.Seeder().Call()` directly in a seeder.

Reply:
We are using `Called` array to keep track of called seeder for CallOnce function

Reply:
The `Called` can be moved to `SeederFacade `.